packages/aik-route-config/contextResolution.ts

```diff
@@ -81,7 +83,7 @@ const integration = defineIntegration({
 
 				// Import SSR components so the hoisted logic gets executed
 				for (const module of ssrComponents) {
-					await import(/* @vite-ignore */ module!);
+					await import(/* @vite-ignore */ module!).catch(() => {});
```

Comment:
Why would this ever fail?

Reply:
There are a few scenarios:
- If the SSR module imports something runtime-specific it will fail since the build runs on Node, not on the SSR runtime.
- If the SSR module imports a TS module, which can run any arbitrary code during import and as such can fail.
- If the SSR module was bundled but then entirely removed by Vite, either because all of it was inlined or because it was deemed unnecessary by tree-shaking, depending on which phase that happens the file is in the list here even though it doesn't exist.